### ISSUE TITLE
SSH Tunnel Port Forwarding 

### ISSUE THREAD

**Issue by nep2ner** [gujjwal00/avnc#2]
First of all great app, I must say that this is the best VNC experience I've had yet on Andriod! 

The integrated SSH tunnel is a great feature. However I noticed that there is no place to specify particular port forwards. How does this work? Does AVNC assume that you want to forward the same remote port as the local port that you specific in AVNC? 

Some users may want to be able to forward remote port X to local port Y. Maybe this functionality could be enabled in settings somewhere? Or maybe I am overlooking it?

Thank you and great work!

**Comment by gujjwal00** [comment]
> First of all great app, I must say that this is the best VNC experience I've had yet on Andriod!

Really glad you like it!

>The integrated SSH tunnel is a great feature. However I noticed that there is no place to specify particular port forwards. How does this work? Does AVNC assume that you want to forward the same remote port as the local port that you specific in AVNC?
>
>Some users may want to be able to forward remote port X to local port Y. Maybe this functionality could be enabled in settings somewhere? Or maybe I am overlooking it?

Can you please explain where/how do you want to setup port forwards?
For now, this is how SSH tunnel works in AVNC:

Without tunnel:
```
[vnc_client:random_port]   <================== Network ==================>    [vnc_server:5900]
```

With tunnel:
```
[vnc_client:random_port]  <---->  [ssh_client:random_port]   <============== Network ==============>    [ssh_server:22] <----> [vnc_server:5900]
```

- Both vnc_client & ssh_client are part of AVNC
- You can specify different host/ports for ssh_server & vnc_server

**Comment by gujjwal00** [comment]
@nep2ner have you resolved this issue?

**Comment by nep2ner** [comment]
Thank you for the detailed explanation @gujjwal00! I misunderstood the `random_port` aspect of this process, but that seems like a great way to do it. Keep up the amazing work!

**Comment by gujjwal00** [comment]
Yeah that could have been explained better. By `random_port` I was referring to a port automatically picked by the system. 

And because SSH client is part of AVNC, it knows what port the system picked. So you don't have to explicitly set the local port for the tunnel.

PS: When I was writing the first comment, I realized that SSH tunnel created by AVNC could accept new connections during the entire VNC session. This has been fixed in the latest release. So thank you for opening this issue.
